backend/src/libs/packages/s3/s3.package.ts

```diff
@@ -16,14 +16,14 @@ import { type ValueOf } from '#libs/types/types.js';
 import { SEC_IN_HOUR } from './libs/constants/constants.js';
 import { type AWSFileUploadRequestDto } from './libs/types/types.js';
 
-type AWSServiceDependencies = {
+type S3PackageDependencies = {
   region: string;
   accessKeyId: string;
   secretAccessKey: string;
   bucketName: string;
 };
 
-class AWSService {
+class S3Package {
```

Comment:
just s3 without package
```suggestion
class S3 {
```